.travis.yml

```diff
@@ -2,24 +2,38 @@ sudo: required
 dist: xenial
 language: python
 cache: pip
+env:
+- TEST_CMD="coverage run tests/test_black.py"
 install:
 - pip install coverage coveralls flake8 flake8-bugbear mypy
```

Comment:
We could speed up builds slightly by only installing what we need, but Travis is pretty fast already and most of this is probably cached anyway, so I don't think it's really worth doing.